### DISCUSSION TITLE
Fine-Tuning Conformer CTC Model

### DISCUSSION THREAD

**Discussion by AdityaYadavalli1** [NVIDIA-NeMo/NeMo#5618] [Q&A] [answered]
Hi, I am new to NeMo and I've been trying to fine-tune the `stt_en_conformer_ctc_large` model on the common voice Japanese dataset by following [this](https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/ASR_CTC_Language_Finetuning.ipynb) tutorial. Instead of loading `stt_en_citrinet_512` as shown in the Subword Encoding CTC Model section, I changed it to `stt_en_conformer_ctc_large`.  After this change, when I run `model.setup_training_data(cfg.train_ds)`, I get the following error:

`manifest_filepaths (length=1) and tarred_audio_filepaths (length=8) need to have the same number of buckets.`

How do I fix this?

**Comment by titu1994** [comment, marked as answer]
I think the tutorial prints model.cfg.train_ds, so just lookup if it has us_tarred and set that to false

**Reply by AdityaYadavalli1** [reply]
Thanks! It was a silly typo on my end but you pointed in the direction.
